Tasks/Voleyko/homework1_deposit.py

```diff
@@ -0,0 +1,11 @@
+import decimal
+from decimal import Decimal
```

Comment:
excess imports, the second line would be enough, the first one could be skipped completely